port_versions/l-/libmariadb.json

```
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "a5144f1b965487813bfd011e1d7ca18877d2ec69",
+      "git-tree": "5748be8cb45e4894504bcac0d47a277efe36f89f",
       "version-string": "3.1.10",
       "port-version": 4
```

```suggestion
      "port-version": 5
```
I'm also wondering how you get git-tree value.